config/default.yml

```diff
@@ -5379,6 +5379,11 @@ Style/TernaryParentheses:
     - require_parentheses_when_complex
   AllowSafeAssignment: true
 
+Style/ToJson:
+  Description: 'Standardize on `to_json` over `JSON.generate`.'
+  Enabled: false
```

Comment:
Do we mean for this to be disabled by default, or do we think this is something the community could agree on (and therefore should be `pending`, to be enabled by default in the next major release, IIRC)?

Reply:
I'd rather not become blocked on the inevitable bikeshed 😆

Reply:
Fair. It can always be changed from `false` to `pending` in a follow up PR.